libs/sdk-ui-kit/src/Dialog/ShareDialog/test/shareDialogMappers.test.ts

```diff
@@ -173,6 +186,94 @@ describe("shareDialogMappers", () => {
         );
     });
 
+    describe("mapWorkspaceUserToGrantee", () => {
+        it("should return correctly map workspace user to grantee", () => {
```

Comment:
"correctly mapped workspace" -> please change map to mapped in other test case names as well.

Reply:
done